### PR TITLE
CI.yml still has some typos

### PR DESCRIPTION
**Pull request by hazel-sudz**
Noticed these while working on a draft for test coverage upload. 

@johnynek @nownikhil

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
             script: './scripts/run_test.sh'
           - scala-version: "2.12.14"
             BUILD: "base"
-            TEST_TAGET: "scalding-avro scalding-hraven scalding-commons scalding-parquet scalding-parquet-scrooge"
+            TEST_TARGET: "scalding-avro scalding-hraven scalding-commons scalding-parquet scalding-parquet-scrooge"
```
Comment (hazel-sudz) [inline .github/workflows/CI.yml:39]: typo here wasn't running these at all

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -86,9 +86,11 @@ jobs:
 
           - scala-version: "2.11.12"
             BUILD: "test tutorials and matrix tutorials and repl"
+            TEST_TARGET: "scalding-repl"
             script: "./scripts/run_test.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_tutorials.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_matrix_tutorials.sh"
           - scala-version: "2.12.14"
             BUILD: "test tutorials and matrix tutorials and repl"
+            TEST_TARGET: "scalding-repl"
             script: "./scripts/run_test.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_tutorials.sh && ./scripts/build_assembly_no_test.sh scalding-assembly && ./scripts/test_matrix_tutorials.sh"
```
Comment (hazel-sudz) [inline .github/workflows/CI.yml:94]: TEST_TARGET was not set compared to old reference travis CI

**Review by johnynek — APPROVED** [review]

**Comment by johnynek** [comment]
Thanks!

**Comment by hazel-sudz** [comment]
github actions has been janky these last couple of days (https://www.githubstatus.com/history). I'll try to push an empty commit to get it to run here

**Comment by johnynek** [comment]
I'm confused why no actions have run.

**Comment by hazel-sudz** [comment]
> I'm confused why no actions have run.

Looks like it's running now...

Github has had a major outage for 48 hours so there is presumably a massive backlog that is still being processed globally.

**Comment by hazel-sudz** [comment]
@johnynek the build finished on the last commit seems good

**Comment by johnynek** [comment]
Thanks again!

**Pull request merged**